.github/workflows/snapshot-record.yaml

```diff
@@ -0,0 +1,32 @@
+name: Snapshot-Record
```

Comment:
This action runs and uploads the pictures to simple storage of github. You can download the pictures in the run. Documentation: https://github.com/actions/upload-artifact

Reply:
Developers can run this an include the pictures in their PR.